dev/tidb-in-kubernetes/deploy/gcp-gke.md

```diff
@@ -507,30 +540,30 @@ If you no longer need the data and would like to delete the disks in use, there
     ```
 
-This command will get the persistent volume claims (PVCs) in the TiDB cluster namespace and set the reclaiming policy of the persistent volumes to `Delete`. When the PVCs are deleted during `terraform destroy`, the disks are deleted as well.
+    This command gets the persistent volume claims (PVCs) in the TiDB cluster namespace and sets the reclaiming policy of the persistent volumes to `Delete`. When the PVCs are deleted during `terraform destroy` execution, the disks are deleted as well.
 
-Below is a script called `change-pv-reclaimpolicy.sh`. It simplifies the above process in `deploy/gcp` directory comparing to the root directory of the repository.
+    The following `change-pv-reclaimpolicy.sh` script simplifies the above process in the `deploy/gcp` directory comparing to the root directory of the repository.
 
-```bash
-./change-pv-reclaimpolicy.sh /path/to/kubeconfig/file tidb-cluster-namespace
-```
+    ```bash
+    ./change-pv-reclaimpolicy.sh /path/to/kubeconfig/file tidb-cluster-namespace
+    ```
 
 ## Manage multiple Kubernetes clusters
 
-This section describes the best practice to manage multiple Kubernetes clusters, each with one or more TiDB clusters installed.
+This section describes the best practices for managing multiple Kubernetes clusters, each with one or more TiDB clusters installed.
 
-The Terraform module in our case typically combines several sub-modules:
+The Terraform module in TiDB typically combines the following sub-modules:
 
-- `tidb-operator`, which provisions the Kubernetes control plane for TiDB clusters
-- `tidb-cluster`, which creates the resource pool in the target Kubernetes cluster and deploys the TiDB cluster
-- A `vpc` module, a `bastion` module and a `project-credentials` module that are dedicated to TiDB on GKE
+- `tidb-operator`: provisions the [Kubernetes Control Plane](https://kubernetes.io/docs/concepts/#kubernetes-control-plane) for TiDB clusters
+- `tidb-cluster`: creates the resource pool in the target Kubernetes cluster and deploys the TiDB cluster
+- A `vpc` module, a `bastion` module, and a `project-credentials` module: dedicated to TiDB clusters on GKE
 
-The best practices for managing multiple Kubernetes clusters are:
+The best practices for managing multiple Kubernetes clusters are as follows:
 
-- Creating a new directory for each of your Kubernetes clusters, and
+- Creating a new directory for each of your Kubernetes clusters.
 - Combining the above modules according to your needs via Terraform scripts.
 
-In this case, the Terraform states among clusters do not interfere with each other, and it is convenient to expand. Here's an example (assume you are in the project root directory):
+If you use the best practices, the Terraform states among clusters do not interfere with each other, and it is convenient to scale out. Here's an example (assume you are in the project root directory):
```

Comment:
The original "expand" means "manage more clusters" instead of "scale out".

Reply:
@DanielZhangQD Got it! Thanks for your explanation.